docker-compose.yml

```diff
@@ -34,15 +34,7 @@ x-snuba-defaults: &snuba_defaults
     CLICKHOUSE_HOST: clickhouse
     DEFAULT_BROKERS: 'kafka:9092'
     REDIS_HOST: redis
-    # TODO: Remove these after getsentry/snuba#353
     UWSGI_MAX_REQUESTS: '10000'
-    UWSGI_DISABLE_LOGGING: 'true'
```

Comment:
This is up to you if you want logging or not, but default is going to bring back logging. So if you want to keep logging disabled, then keep this one. I personally thing the logging is useful in production instances.

Reply:
I'll keep logging disabled for now as it may become noisy and cause people's disks to fill up.